solutions/swb-reference/README.md

```diff
@@ -6,348 +6,737 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
+      "value": "0"
+     },
+     {
+       "key": "CIDR",
+       "value": "0.0.0.0/0"
+     }
+    ]
+}
+```
+In the response take note of the `id` that was returned. We'll refer to this value as `ENV_TYPE_CONFIG_ID`.
 
-    1. Create an environment by opening Postman Collection and select `Launch Environment` inside `environments` folder
-    
-    2. Click on the Body section of the request and fill the next values 
-        ```
-        {
-            "description": "test 123",
-            "name": "testEnv1",
-            "envTypeId": "<environment type id>",
-            "envTypeConfigId": "<environment type config id>",
-            "projectId": "<project id>",
-            "datasetIds": [],
-            "envType": "<environemnt type>"
-        }
-        ```
-        We can use the values from previous steps to fill the create request body
+If you would like to launch a sagemaker notebook instance with a different instance type than `ml.t3.medium`, you can replace that value in the JSON above.
 
-    3. Execute `Launch Environment` request and copy the id property from the response
-        ```
-        {
-            "id": "########-####-####-####-############",
-            "instanceId": "",
-            "cidr": "",
-            "description": "description",
-            "name": "environment Name",
-            "outputs": [],
-            ......
-        ```
+### Associate Project to Environment Type Configuration
+In **SWBv2 Official** Postman Collection under **project** folder choose **Associate project with EnvTypeConfig** API.
+
+In the params tab set `projectId` parameter to the `PROJECT_ID` value from [Setup Project step](#setup-project).
+
+In the body tab set `envTypeId` parameter to the `ENV_TYPE_ID` value from [Retrieve Environment Type Id step](#retrieve-environment-type-id).
+
+In the body tab set `envTypeConfigId` parameter to the `ENV_TYPE_CONFIG_ID` value from [Create Environment Type Config step](#create-environment-type-config).
+
+Send **Associate project with EnvTypeConfig** request.
+
+PUT `{{API_URL}}/projects/:projectId/environmentTypes/:envTypeId/configurations/:envTypeConfigId/relationships`
+
+
+## Create new DataSet
+In **SWBv2 Official** Postman Collection under **datasets** folder choose **Create Internal DataSet** API.
+
+In the params tab set `projectId` parameter to the `PROJECT_ID` value from [Setup Project step](#setup-project).
+
+In the body tab set `region` parameter to the `awsRegion` value from [Setup Config File Step](#setup-config-file).
+
+In the body tab set `storageName` parameter to the `DataSetsBucketName` value from [Deploy The Code Step](#deploy-the-code).
+
+Note: Only Researchers and Project Admins can access this API, the user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+POST `{{API_URL}}/projects/:projectId/datasets/`
+
+```json
```

Comment:
Maybe a broader discussion, but do we even need to provide these snippets if the postman collection provides a sample body anyways?

Reply:
IMO It helps users know they are in the right API request , or in case they miss click another API that may help them realize it

Reply:
Fair enough.